lesson_03/homework_03.py

```diff
@@ -0,0 +1,218 @@
+                       '"Oh, you\'re sure to do that," '
+                       'said the Cat, "if you only walk long enough."')
+if alice_in_wonderland.count("'") >= 2:
+    print('There are', alice_in_wonderland.count("'"), 'single quotes present')
+elif alice_in_wonderland.count("'") == 1:
+    print('There is', alice_in_wonderland.count("'"), 'single quote present')
+else:
+    print('Single quotes are missing in the text.')
+
+
+""" Task 3 + code
+# task 03 == Виведіть змінну alice_in_wonderland на друк """
+
+
+alice_in_wonderland = ('"Would you tell me, please, which way I '
+                       'ought to go from here?"\n'
+                       '"That depends a good deal on where you '
+                       'want to get to," said the Cat.\n'
+                       '"I don\'t much care where ——" said Alice.\n'
+                       '"Then it doesn\'t matter which way you go," '
+                       'said the Cat.\n'
+                       '—— so long as I get somewhere," '
+                       'Alice added as an explanation.\n'
+                       '"Oh, you\'re sure to do that," said the Cat, '
+                       '"if you only walk long enough."')
+
+print(alice_in_wonderland)
+
+
+""" Tasks 4-10 description
+    # Задачі 04 -10:
+    # Переведіть задачі з книги "Математика, 5 клас"
+    # на мову пітон і виведіть відповідь, так, щоб було
+    # зрозуміло дитині, що навчається в п'ятому класі
+"""
+
+"""
+Task 4 + code
+Площа Чорного моря становить 436 402 км2, а площа Азовського
+моря становить 37 800 км2. Яку площу займають Чорне та Азов-
+ське моря разом?
+"""
+black_sea_square = 436402
+azov_sea_square = 37800
+two_seas_square = black_sea_square + azov_sea_square
+print('The total square for Azov and Black seas is:', two_seas_square)
+
+"""
+Task 5 + code
+Мережа супермаркетів має 3 склади, де всього розміщено
+375 291 товар. На першому та другому складах перебуває
+250 449 товарів. На другому та третьому – 222 950 товарів.
+Знайдіть кількість товарів, що розміщені на кожному складі.
+"""
+
+total_goods = 375291
+fir_and_sec_w = 250449
+sec_and_thi_w = 222950
+sec_war = fir_and_sec_w + sec_and_thi_w - total_goods
+fir_war = fir_and_sec_w - sec_war
+thi_war = sec_and_thi_w - sec_war
+print('The first warehouse has:', fir_war, 'goods\n'
+      'The second warehouse has:', sec_war, 'goods\n'
+      'The third warehouse has:', thi_war, 'goods')
+
+"""
+Task 6 + code
+Михайло разом з батьками вирішили купити комп’ютер, ско-
+риставшись послугою «Оплата частинами». Відомо, що сплачу-
+вати необхідно буде півтора року по 1179 грн/місяць. Обчисліть
+вартість комп’ютера.
+"""
+
+price_per_month = 1179
+months = 18
+total_value = price_per_month * months
+print('The total PC price is:', total_value)
+
+
+"""
+Task 7 + code
+Знайди остачу від діленя чисел:
+a) 8019 : 8     d) 7248 : 6
+b) 9907 : 9     e) 7128 : 5
+c) 2789 : 5     f) 19224 : 9
+"""
+
+a = 8019 % 8
+print('The remainder for a is: ', a)
```

Comment:
```suggestion
print(f' a) Answer: {8019%8} ')
```
Not critical - just could be simplified.
We will learn f-strings usage next time - just pay attention that they are more compact and more powerful